cfme/tests/services/test_dialog_element_in_catalog.py

```diff
@@ -1003,20 +1003,16 @@ def test_dialog_dropdown_integer_required():
         casecomponent: Services
         initialEstimate: 1/16h
         startsin: 5.10
-        testSteps:
```

Comment:
Why did you remove testSteps and expectedResults?

Reply:
added